### PR TITLE
Feature/7649 adjust info screen to announce new release features for 2.5

### PR DESCRIPTION
**Pull request by SwissMountainFrog**
This PR contains the new version features info screen for version 2.5

Ticket https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7649

IMG_0594
IMG_0595

### CODE REVIEW

**Review by KaiTeuber — APPROVED** [review]

**Comment by ChristinaTsigganos** [comment]
Texts have been adjusted because features for 2.5 have been reduced. New texts for the "Neue Funktionen"-screen can be found here: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7649 Can you please adapt? Thanks! @SwissMountainFrog @InkenTownsend FYI

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 1]
```diff
@@ -1120,15 +1120,23 @@ enum AppStrings {
...
+		static let feature25RecoveryCertTitle = NSLocalizedString("NewVersionFeature_25_recovery_cert_title", comment: "")
 		
-		static let feature24digitalCovidCertDescription = NSLocalizedString("NewVersionFeature_24_digital_covid_cert_description", comment: "")
+		static let feature25tRecoveryCertDescription = NSLocalizedString("NewVersionFeature_25_recovery_cert_description", comment: "")
 		
-		static let feature24ExtendedDiaryTitle = NSLocalizedString("NewVersionFeature_24_extended_diary_title", comment: "")
+		static let feature25FamilyCertTitle = NSLocalizedString("NewVersionFeature_25_family_certificate_title", comment: "")
 		
-		static let feature24ExtendedDiaryDescription = NSLocalizedString("NewVersionFeature_24_extended_diary_title_description", comment: "")
+		static let feature25tFamilyCertDescription = NSLocalizedString("NewVersionFeature_25_family_certificate_description", comment: "")
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:1131]:
```suggestion
		static let feature25FamilyCertDescription = NSLocalizedString("NewVersionFeature_25_family_certificate_description", comment: "")
```

Comment (pascal-brause) [reply src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:1131]: there is 't' too much.

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/NewVersionFeatures/DeltaOnboardingNewVersionFeaturesViewModel.swift [hunk 2]
```diff
@@ -13,18 +13,30 @@ struct DeltaOnboardingNewVersionFeaturesViewModel {
...
+		self.newVersionFeatures.append(
+			NewVersionFeature(title: AppStrings.NewVersionFeatures.feature25EnhancedStatisticsTitle, description: AppStrings.NewVersionFeatures.feature25EnhancedStatisticsDescription)
+		)
+		
+		// Edit Test Profile
+		/*
+		self.newVersionFeatures.append(
+			NewVersionFeature(title: AppStrings.NewVersionFeatures.feature25EditTestProfileTitle, description: AppStrings.NewVersionFeatures.feature25tEditTestProfileDescription)
+		)
+		*/
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/NewVersionFeatures/DeltaOnboardingNewVersionFeaturesViewModel.swift:38]: For later?

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/NewVersionFeatures/DeltaOnboardingNewVersionFeaturesViewModel.swift [hunk 3]
```diff
@@ -13,18 +13,30 @@ struct DeltaOnboardingNewVersionFeaturesViewModel {
...
 		self.newVersionFeatures.append(
-			NewVersionFeature(title: AppStrings.NewVersionFeatures.feature24digitalCovidCertTitle, description: AppStrings.NewVersionFeatures.feature24digitalCovidCertDescription)
+			NewVersionFeature(title: AppStrings.NewVersionFeatures.feature25RecoveryCertTitle, description: AppStrings.NewVersionFeatures.feature25tRecoveryCertDescription)
 		)
 		
-		// Extended Diary
+		// Family Certificate
 		self.newVersionFeatures.append(
-			NewVersionFeature(title: AppStrings.NewVersionFeatures.feature24ExtendedDiaryTitle, description: AppStrings.NewVersionFeatures.feature24ExtendedDiaryDescription)
+			NewVersionFeature(title: AppStrings.NewVersionFeatures.feature25FamilyCertTitle, description: AppStrings.NewVersionFeatures.feature25tFamilyCertDescription)
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/NewVersionFeatures/DeltaOnboardingNewVersionFeaturesViewModel.swift:25]:
```suggestion
			NewVersionFeature(title: AppStrings.NewVersionFeatures.feature25FamilyCertTitle, description: AppStrings.NewVersionFeatures.feature25FamilyCertDescription)
```

**Review by whiskey — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift (hunk 1 above)
Comment (SwissMountainFrog) [reply src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:1131]: Thanks man!

**Review by naveeddotio — APPROVED** [review]

**Review by ChristinaTsigganos — APPROVED** [review]
UA approved for now - but the info about "Schnelltest-Profil" has to be removed (Karsten is already aware)

**Pull request merged**